### ISSUE TITLE
Re: accessible validation of checkbox and radio buttons

### ISSUE THREAD

**Issue by JamesCatt** [tenon-io/tenon-ui#49]
I was just reading [your blog post on this subject](https://blog.tenon.io/accessible-validation-of-checkbox-and-radiobutton-groups), but I'm a little confused. Unless I just missed it, you don't appear to have considered just adding `aria-describedby` to all the affected inputs, i.e.,:
```
<fieldset aria-describedby="foo">
    <legend>Select your favourite colours.</legend>
    <input type="checkbox" name="blackInput" id="black" aria-describedby="foo" />
    <label for="black">Black</label>
    <input type="checkbox" name="redInput" id="red" aria-describedby="foo" />
    <label for="red">Red</label>
</fieldset>
<span id="foo">You must select at least one colour</span>
```
Just to be clear—this is perfectly valid and works to announce the message for both inputs, as while `id` attributes must be unique, the `aria-describedby` attribute is only _referring_ to this `id`, and does not need to be unique itself. You can have as many references like this as you want.

Apologies if there's something I'm missing here.
